Ct 1754 send sar to dacu

beccapearce opened the pull request.

misaka commented on spec/controllers/messages_controller_spec.rb:
```diff
@@ -0,0 +1,119 @@
+    context "as Press Office" do
+      before { sign_in press_officer }
+
+      it "doesn't allow them to post messages to non-trigger cases" do
```
Would it make sense to just test that that an approver has to be assigned to the case to be able to add a message to it? Unless the case policy expressly tests for the teams I think that would make sense.

Commit:
CT-1754 add progress for clearance journey

Commit:
CT-1754 add authorise and helper test

Commit:
CT-1754 fix rubocop errors

stephenrichards reviewed (approved):
just two little things, but apart from that very nice.

Commit:
ct-1754 add target team and test controller action

Commit:
CT-1754 Adds progress_for_clearance event

In order for the case transitions decorator tests to pass
the event name needs to be put ion the state machine

Commit:
CT-1754 disables CyclomaticComplexity in transition decorator

beccapearce committed:
Merge branch 'master' into ct-1754-send-sar-to-dacu

beccapearce committed:
Merge branch 'master' into ct-1754-send-sar-to-dacu
